jerry-ext/CMakeLists.txt

```diff
@@ -20,11 +20,12 @@ project (${JERRY_EXT_NAME} C)
 set(INCLUDE_EXT "${CMAKE_CURRENT_SOURCE_DIR}/include")
 
 # Source directories
-file(GLOB SOURCE_EXT arg/*.c)
+file(GLOB SOURCE_EXT arg/*.c context/*.c)
 
 add_library(${JERRY_EXT_NAME} STATIC ${SOURCE_EXT})
 
 target_include_directories(${JERRY_EXT_NAME} PUBLIC ${INCLUDE_EXT})
+target_include_directories(${JERRY_EXT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/private-headers")
```

Comment:
Sorry, I don't understand the answer. "Yes", it is needed, or "yes", it could go into the context directory?